### PR TITLE
SceneRPCInterface: Sort rpc names using `StringLikeVariantOrder`

### PR DESCRIPTION
**Pull request by rune-scape**

### CODE REVIEW

File: modules/multiplayer/scene_rpc_interface.cpp [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 #include "scene_rpc_interface.h"
 
+#include "core/variant/variant.h"
```
Comment (AThousandShips) [inline modules/multiplayer/scene_rpc_interface.cpp:33]:
```suggestion
```
Already included

**Review by AThousandShips — APPROVED** [review]
Tentatively approving, the drawback is potential loss of performance by not having a `likely`

**Comment by clayjohn** [comment]
Is the goal of this PR just to avoid code duplication? Or is there an intended behaviour difference in this sorting function?

**Comment by rune-scape** [comment]
just to avoid code duplication

**Review by akien-mga — APPROVED** [review]

**Pull request merged**